src/components/buttons/SecondaryBtn.tsx

```diff
@@ -0,0 +1,33 @@
+import { Button } from '@mui/material'
+import { ReactNode } from 'react'
+
+export const SecondaryBtn = ({
+  startIcon,
+  buttonContent,
+  handleClick,
+}: {
+  startIcon?: ReactNode | undefined
```

Comment:
the prop is already optional, do we need extra undefined type?

Reply:
removed